data/json/items/ammo/pebble.json

```diff
@@ -0,0 +1,49 @@
+[
+  {
+    "id": "bearing",
+    "copy-from": "pebble",
```

Comment:
What exactly is being copied from here?
You need to use `relative` or `proportional` for `range`, `damage` etc